views/js/helper/status.js

```diff
@@ -38,7 +38,10 @@ define(['lodash', 'i18n'], function(_, __){
                 print : __('not finished'),
                 reactivate : __('not terminated'),
                 time : function(delivery) {
-                    return delivery.timer.remaining_time > 0;
+                    if (delivery.timer.remaining_time > 0) {
```

Comment:
1. (not principle):
```js
     return (delivery.timer.remaining_time > 0) || __('test has no time limits');
```
2. (principle) Since the time function is duplicated 4 times, I suggest to declare it and reuse